Userland/Libraries/LibGUI/AbstractZoomPanWidget.cpp

```diff
@@ -180,4 +180,29 @@ void AbstractZoomPanWidget::set_scale_bounds(float min_scale, float max_scale)
     m_max_scale = max_scale;
 }
 
```

Comment:
The comment message says that we're _moving_ this functionality, but there's no moving happening in this commit at all!